autoload/neomake/makers/ft/elixir.vim

```diff
@@ -7,6 +7,23 @@ function! neomake#makers#ft#elixir#PostprocessEnforceMaxBufferLine(entry) abort
     endif
 endfunction
 
+function! neomake#makers#ft#elixir#PostprocessCredoErrorType(entry) abort
+    if a:entry.type ==# 'F'      " Refactoring opportunities
+        let type = 'W'
+    elseif a:entry.type ==# 'D'  " Software design suggestions
+        let type = 'I'
+    elseif a:entry.type ==# 'W'  " Warnings
+        let type = 'W'
+    elseif a:entry.type ==# 'R'  " Readability suggestions
+        let type = 'I'
+    elseif a:entry.type ==# 'C'  " Convention violation
+        let type = 'W'
+    else
+        let type = 'M'           " Everything else is a message
+    endif
+    let a:entry.type = type
+endfunction
```

Comment:
in closer inspection, credo shouldn't give us back (at least by default) anything outside of those 5 groups. I will remove that part.
Then, I was thinking that perhaps users might have different ideas of what a translation should be. so I've created a configurable variable to override the defaults. 
Please let me know what you think